components/payments/cmd/connectors/internal/connectors/atlar/task_fetch_transactions.go

```diff
@@ -91,9 +93,20 @@ func ingestPaymentsBatch(
 }
 
 func atlarTransactionToPaymentBatchElement(
+	ctx context.Context,
 	connectorID models.ConnectorID,
+	taskID models.TaskID,
 	transaction *atlar_models.Transaction,
+	client *client.Client,
 ) (*ingestion.PaymentBatchElement, error) {
+	ctx, span := connectors.StartSpan(
+		ctx,
+		"atlar.atlarTransactionToPaymentBatchElement",
+		attribute.String("connectorID", connectorID.String()),
+		attribute.String("taskID", taskID.String()),
+	)
+	defer span.End()
```

Comment:
I hope creating instead of passing a span is the correct thing to do here.

Reply:
Yep it's good, it's gonna be linked to the parent span thanks to the StartSpan function, so LGTM